### PR TITLE
Add item in Spinner

### PR DESCRIPTION
**Pull request by JuSeongPil**
thank u everyone

### CODE REVIEW

**Review by jopopcorn — CHANGES_REQUESTED** [review]
Please correct the part I reviewed. Please cheer up a little bit more!

File: app/src/main/res/layout/tutorial.xml [hunk 1]
```diff
@@ -47,14 +44,31 @@
         android:layout_marginLeft="20dp"
         android:hint="Gangnam-gu, Seoul"
         android:inputType="textCapSentences" />
-
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
```
Comment (sujin9) [inline app/src/main/res/layout/tutorial.xml:50]:
```suggestion
        android:orientation="horizontal"
        android:padding="10dp" >
```

**Review by sujin9 — COMMENTED** [review]
Good job !
Same opinion as Sooyeon :)
Also you should delete livingCity EditText later cuz we're gonna use Spinner instead right :)

File: app/src/main/res/layout/tutorial.xml (hunk 1 above)
Comment (sujin9) [reply app/src/main/res/layout/tutorial.xml:50]: And one more suggestion here! 
It looks better if there's a bit of white space.

File: app/src/main/res/values/array.xml [hunk 2]
```diff
@@ -0,0 +1,845 @@
...
+        <item>강일동</item>
+        <item>고덕동</item>
+        <item>길동</item>
+        <item>둔촌동</item>
+        <item>명일동</item>
+        <item>상일동</item>
+        <item>성내동</item>
+        <item>암사동</item>
+        <item>천호동</item>
+    </string-array>
```
[the remark this answers is not in the document]
Comment (JuSeongPil) [reply app/src/main/res/values/array.xml:79]: hi, it is a array.xml file?
but i just using strings.xml file in this project! 
thank u!

**Pull request closed without merging**